client/src/components/AuthenticationForm.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import Input from './Input';
+import Button from './Button';
+import styled from '@emotion/styled';
+import colors from '../utils/colorsUtils';
+import PropTypes from 'prop-types';
+
+const ButtonContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  > * {
+    flex-basis: 45%;
+  }
+`;
+
+const InputContainer = styled.div`
+  border: solid 2px ${colors.primaryYellow};
+  border-radius: 24px;
+  background-color: ${colors.primaryDark};
```

Comment:
A very similar argument like the above comment:
Maybe call it `colors.background`?